[2024-05-05T14:04:23.362+0000] {processor.py:161} INFO - Started process (PID=13627) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:04:23.362+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:04:23.364+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:04:23.364+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:04:23.368+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:04:23.474+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:04:23.473+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:hello
[2024-05-05T14:04:23.491+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:04:23.491+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:hello
[2024-05-05T14:04:23.502+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:04:23.502+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:hello
[2024-05-05T14:04:23.502+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:04:23.502+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:04:23.513+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:04:23.513+0000] {dag.py:3118} INFO - Creating ORM DAG for hello
[2024-05-05T14:04:23.514+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:04:23.514+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:04:23.535+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.179 seconds
[2024-05-05T14:04:53.933+0000] {processor.py:161} INFO - Started process (PID=13899) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:04:53.934+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:04:53.936+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:04:53.936+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:04:53.940+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:04:53.968+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:04:53.968+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:04:53.981+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:04:53.981+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:04:54.003+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.077 seconds
[2024-05-05T14:05:24.711+0000] {processor.py:161} INFO - Started process (PID=14171) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:05:24.712+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:05:24.713+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:05:24.713+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:05:24.717+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:05:24.733+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:05:24.733+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:05:24.744+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:05:24.744+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:05:24.769+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.063 seconds
[2024-05-05T14:05:55.450+0000] {processor.py:161} INFO - Started process (PID=14444) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:05:55.451+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:05:55.452+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:05:55.452+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:05:55.457+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:05:55.476+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:05:55.476+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:05:55.487+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:05:55.486+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:05:55.508+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.067 seconds
[2024-05-05T14:06:26.147+0000] {processor.py:161} INFO - Started process (PID=14717) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:06:26.148+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:06:26.150+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:06:26.150+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:06:26.156+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:06:26.178+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:06:26.178+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:06:26.197+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:06:26.197+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:06:26.222+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.082 seconds
[2024-05-05T14:06:56.980+0000] {processor.py:161} INFO - Started process (PID=14990) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:06:56.981+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:06:56.982+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:06:56.982+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:06:56.987+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:06:57.007+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:06:57.007+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:06:57.024+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:06:57.024+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:06:57.052+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.080 seconds
[2024-05-05T14:07:27.573+0000] {processor.py:161} INFO - Started process (PID=15263) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:07:27.574+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:07:27.575+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:07:27.575+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:07:27.579+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:07:27.594+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:07:27.594+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:07:27.605+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:07:27.605+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:07:27.625+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.058 seconds
[2024-05-05T14:07:58.257+0000] {processor.py:161} INFO - Started process (PID=15536) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:07:58.258+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:07:58.259+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:07:58.259+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:07:58.262+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:07:58.278+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:07:58.277+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:07:58.288+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:07:58.288+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:07:58.310+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.057 seconds
[2024-05-05T14:08:29.063+0000] {processor.py:161} INFO - Started process (PID=15808) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:08:29.064+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:08:29.065+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:08:29.065+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:08:29.068+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:08:29.084+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:08:29.084+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:08:29.094+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:08:29.094+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:08:29.117+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.059 seconds
[2024-05-05T14:08:59.664+0000] {processor.py:161} INFO - Started process (PID=16081) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:08:59.665+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:08:59.666+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:08:59.666+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:08:59.670+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:08:59.686+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:08:59.686+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:08:59.698+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:08:59.698+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:08:59.719+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.060 seconds
[2024-05-05T14:09:29.962+0000] {processor.py:161} INFO - Started process (PID=16352) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:09:29.962+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:09:29.964+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:09:29.963+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:09:29.967+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:09:29.983+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:09:29.983+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:09:29.994+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:09:29.994+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:09:30.015+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.057 seconds
[2024-05-05T14:10:00.147+0000] {processor.py:161} INFO - Started process (PID=16624) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:00.148+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:00.149+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:00.149+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:00.152+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:00.171+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:00.170+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:10:00.182+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:00.181+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:10:00.204+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.061 seconds
[2024-05-05T14:10:06.050+0000] {processor.py:161} INFO - Started process (PID=16679) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:06.051+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:06.053+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:06.052+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:06.057+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:06.204+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:06.204+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:10:06.218+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:06.218+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:10:06.250+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.207 seconds
[2024-05-05T14:10:08.852+0000] {processor.py:161} INFO - Started process (PID=16725) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:08.852+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:08.855+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:08.855+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:08.863+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:08.859+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 14, in <module>
    my_task >> my_task_II
    ^^^^^^^
NameError: name 'my_task' is not defined. Did you mean: 'my_task_II'?
[2024-05-05T14:10:08.866+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:08.900+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.056 seconds
[2024-05-05T14:10:11.402+0000] {processor.py:161} INFO - Started process (PID=16770) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:11.402+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:11.404+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:11.403+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:11.409+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:11.407+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 14, in <module>
    my_task >> my_task_II
    ^^^^^^^
NameError: name 'my_task' is not defined
[2024-05-05T14:10:11.411+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:11.435+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.039 seconds
[2024-05-05T14:10:14.532+0000] {processor.py:161} INFO - Started process (PID=16835) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:14.533+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:14.535+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:14.535+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:14.544+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:14.540+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 14, in <module>
    my_task >> my_task_II
    ^^^^^^^
NameError: name 'my_task' is not defined
[2024-05-05T14:10:14.548+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:14.577+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.052 seconds
[2024-05-05T14:10:17.652+0000] {processor.py:161} INFO - Started process (PID=16850) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:17.653+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:17.654+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:17.654+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:17.660+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:17.657+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 17, in <module>
    my_task >> my_task_II
    ^^^^^^^
NameError: name 'my_task' is not defined
[2024-05-05T14:10:17.662+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:17.685+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.039 seconds
[2024-05-05T14:10:22.683+0000] {processor.py:161} INFO - Started process (PID=16855) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:22.683+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:22.685+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:22.684+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:22.692+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:22.688+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 17, in <module>
    start >> my_task_II
             ^^^^^^^^^^
NameError: name 'my_task_II' is not defined
[2024-05-05T14:10:22.696+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:22.718+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.041 seconds
[2024-05-05T14:10:23.738+0000] {processor.py:161} INFO - Started process (PID=16860) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:23.739+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:23.741+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:23.741+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:23.747+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:23.762+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:23.762+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:10:23.781+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:23.780+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:10:23.818+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.087 seconds
[2024-05-05T14:10:31.128+0000] {processor.py:161} INFO - Started process (PID=16931) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:31.130+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:31.131+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:31.131+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:31.136+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:31.151+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:31.151+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:10:31.170+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:31.170+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:10:31.200+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.078 seconds
[2024-05-05T14:10:33.341+0000] {processor.py:161} INFO - Started process (PID=16951) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:33.342+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:33.344+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:33.343+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:33.351+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:33.349+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    B
NameError: name 'B' is not defined
[2024-05-05T14:10:33.352+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:33.381+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.047 seconds
[2024-05-05T14:10:34.485+0000] {processor.py:161} INFO - Started process (PID=16966) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:34.487+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:34.489+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:34.489+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:34.496+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:34.494+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    Bas
NameError: name 'Bas' is not defined
[2024-05-05T14:10:34.498+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:34.536+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.058 seconds
[2024-05-05T14:10:36.897+0000] {processor.py:161} INFO - Started process (PID=16996) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:36.898+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:36.900+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:36.900+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:36.907+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:36.905+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    Bash
NameError: name 'Bash' is not defined. Did you mean: 'hash'?
[2024-05-05T14:10:36.909+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:36.941+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.051 seconds
[2024-05-05T14:10:38.201+0000] {processor.py:161} INFO - Started process (PID=17022) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:38.202+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:38.204+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:38.204+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:38.212+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:38.209+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOp
NameError: name 'BashOp' is not defined
[2024-05-05T14:10:38.215+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:38.264+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.070 seconds
[2024-05-05T14:10:42.107+0000] {processor.py:161} INFO - Started process (PID=17087) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:42.107+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:42.109+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:42.108+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:42.114+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:42.112+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOper
NameError: name 'BashOper' is not defined
[2024-05-05T14:10:42.116+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:42.141+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.040 seconds
[2024-05-05T14:10:45.859+0000] {processor.py:161} INFO - Started process (PID=17157) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:45.860+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:45.862+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:45.861+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:45.885+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:45.874+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator
NameError: name 'BashOperator' is not defined
[2024-05-05T14:10:45.905+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:45.930+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.077 seconds
[2024-05-05T14:10:47.976+0000] {processor.py:161} INFO - Started process (PID=17167) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:47.976+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:47.978+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:47.977+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:47.985+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:47.981+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator()
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:10:47.988+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:48.011+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.041 seconds
[2024-05-05T14:10:48.982+0000] {processor.py:161} INFO - Started process (PID=17172) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:48.982+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:48.984+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:48.984+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:48.996+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:48.988+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:10:49.002+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:49.025+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.049 seconds
[2024-05-05T14:10:52.065+0000] {processor.py:161} INFO - Started process (PID=17177) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:52.066+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:52.067+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:52.067+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:52.075+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:52.071+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:10:52.080+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:52.104+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.046 seconds
[2024-05-05T14:10:56.161+0000] {processor.py:161} INFO - Started process (PID=17187) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:56.162+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:56.164+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:56.163+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:56.171+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:56.167+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:10:56.176+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:56.198+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.043 seconds
[2024-05-05T14:10:58.235+0000] {processor.py:161} INFO - Started process (PID=17197) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:10:58.236+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:10:58.241+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:58.240+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:10:58.253+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:10:58.246+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:10:58.259+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:10:58.319+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.090 seconds
[2024-05-05T14:11:00.785+0000] {processor.py:161} INFO - Started process (PID=17237) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:00.786+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:00.788+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:00.787+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:00.800+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:00.793+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:00.809+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:00.836+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.058 seconds
[2024-05-05T14:11:06.985+0000] {processor.py:161} INFO - Started process (PID=17313) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:06.986+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:06.987+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:06.987+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:06.995+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:06.991+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:06.998+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:07.021+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.041 seconds
[2024-05-05T14:11:10.333+0000] {processor.py:161} INFO - Started process (PID=17345) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:10.334+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:10.336+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:10.336+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:10.348+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:10.341+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:10.356+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:10.384+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.057 seconds
[2024-05-05T14:11:14.261+0000] {processor.py:161} INFO - Started process (PID=17448) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:14.262+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:14.264+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:14.264+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:14.275+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:14.268+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:14.282+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:14.327+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.072 seconds
[2024-05-05T14:11:22.615+0000] {processor.py:161} INFO - Started process (PID=17480) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:22.616+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:22.617+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:22.617+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:22.625+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:22.621+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:22.628+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:22.651+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.042 seconds
[2024-05-05T14:11:24.799+0000] {processor.py:161} INFO - Started process (PID=17488) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:24.810+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:24.812+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:24.812+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:24.839+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:24.825+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:24.846+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:24.911+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.149 seconds
[2024-05-05T14:11:28.967+0000] {processor.py:161} INFO - Started process (PID=17500) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:28.968+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:28.970+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:28.969+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:28.982+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:28.975+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:28.989+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:29.018+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.058 seconds
[2024-05-05T14:11:31.804+0000] {processor.py:161} INFO - Started process (PID=17559) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:31.805+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:31.807+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:31.807+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:31.820+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:31.812+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:31.827+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:31.856+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.060 seconds
[2024-05-05T14:11:33.166+0000] {processor.py:161} INFO - Started process (PID=17581) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:33.166+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:33.167+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:33.167+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:33.174+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:33.170+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:33.178+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:33.199+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.038 seconds
[2024-05-05T14:11:35.302+0000] {processor.py:161} INFO - Started process (PID=17596) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:35.303+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:35.304+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:35.303+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:35.316+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:35.308+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    BashOperator(
    ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:35.330+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:35.416+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.121 seconds
[2024-05-05T14:11:43.876+0000] {processor.py:161} INFO - Started process (PID=17708) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:43.877+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:43.878+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:43.877+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:43.889+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:43.881+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 13, in <module>
    echo_hello = BashOperator(
                 ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:43.909+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:43.947+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.078 seconds
[2024-05-05T14:11:52.763+0000] {processor.py:161} INFO - Started process (PID=17783) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:52.763+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:52.764+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:52.764+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:52.780+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:52.768+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 14, in <module>
    echo_hello = BashOperator(
                 ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-05-05T14:11:52.788+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:52.812+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.056 seconds
[2024-05-05T14:11:57.856+0000] {processor.py:161} INFO - Started process (PID=17793) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:11:57.857+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:11:57.858+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:57.857+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:11:57.862+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:11:57.860+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 3, in <module>
    from airflow.operators.empty import BashOperator
ImportError: cannot import name 'BashOperator' from 'airflow.operators.empty' (/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/empty.py)
[2024-05-05T14:11:57.864+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:11:57.886+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.034 seconds
[2024-05-05T14:12:02.746+0000] {processor.py:161} INFO - Started process (PID=17857) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:12:02.747+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:12:02.748+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:12:02.748+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:12:02.756+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:12:02.965+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:12:02.965+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:12:02.994+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:12:02.993+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:12:03.033+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.294 seconds
[2024-05-05T14:12:33.442+0000] {processor.py:161} INFO - Started process (PID=18111) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:12:33.443+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:12:33.444+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:12:33.444+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:12:33.451+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:12:33.474+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:12:33.473+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:12:33.490+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:12:33.490+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:12:33.518+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.083 seconds
[2024-05-05T14:13:03.982+0000] {processor.py:161} INFO - Started process (PID=18384) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:13:03.983+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:13:03.984+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:13:03.984+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:13:03.990+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:13:04.014+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:13:04.014+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:13:04.035+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:13:04.035+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:13:04.069+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.093 seconds
[2024-05-05T14:13:34.524+0000] {processor.py:161} INFO - Started process (PID=18656) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:13:34.525+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:13:34.526+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:13:34.526+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:13:34.532+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:13:34.555+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:13:34.555+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:13:34.571+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:13:34.571+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:13:34.599+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.081 seconds
[2024-05-05T14:14:05.422+0000] {processor.py:161} INFO - Started process (PID=18928) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:14:05.423+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:14:05.425+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:14:05.425+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:14:05.431+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:14:05.454+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:14:05.453+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:14:05.470+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:14:05.470+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:14:05.498+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.083 seconds
[2024-05-05T14:14:35.745+0000] {processor.py:161} INFO - Started process (PID=19200) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:14:35.746+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:14:35.749+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:14:35.748+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:14:35.763+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:14:35.815+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:14:35.815+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:14:35.870+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:14:35.870+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:14:35.912+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.175 seconds
[2024-05-05T14:15:06.641+0000] {processor.py:161} INFO - Started process (PID=19473) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:15:06.642+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:15:06.643+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:15:06.643+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:15:06.649+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:15:06.673+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:15:06.672+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:15:06.689+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:15:06.688+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:15:06.717+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.083 seconds
[2024-05-05T14:15:37.251+0000] {processor.py:161} INFO - Started process (PID=19745) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:15:37.252+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:15:37.253+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:15:37.253+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:15:37.259+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:15:37.283+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:15:37.282+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:15:37.299+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:15:37.299+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:15:37.327+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.083 seconds
[2024-05-05T14:16:07.821+0000] {processor.py:161} INFO - Started process (PID=20017) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:16:07.822+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:16:07.823+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:16:07.823+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:16:07.833+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:16:07.857+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:16:07.856+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:16:07.873+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:16:07.873+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:16:07.904+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.089 seconds
[2024-05-05T14:16:38.020+0000] {processor.py:161} INFO - Started process (PID=20288) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:16:38.021+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:16:38.022+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:16:38.022+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:16:38.028+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:16:38.056+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:16:38.056+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:16:38.073+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:16:38.072+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:16:38.110+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.096 seconds
[2024-05-05T14:16:58.064+0000] {processor.py:161} INFO - Started process (PID=20515) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:16:58.065+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:16:58.066+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:16:58.065+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:16:58.068+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:16:58.067+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 23
    start >> echo_hello end
                        ^^^
SyntaxError: invalid syntax
[2024-05-05T14:16:58.069+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:16:58.091+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.032 seconds
[2024-05-05T14:17:00.078+0000] {processor.py:161} INFO - Started process (PID=20520) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:17:00.078+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:17:00.079+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:17:00.079+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:17:00.084+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:17:00.181+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:17:00.180+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:17:00.189+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:17:00.189+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:17:00.223+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.150 seconds
[2024-05-05T14:17:30.507+0000] {processor.py:161} INFO - Started process (PID=20793) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:17:30.507+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:17:30.508+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:17:30.508+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:17:30.513+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:17:30.529+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:17:30.529+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:17:30.540+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:17:30.540+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:17:30.561+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.060 seconds
[2024-05-05T14:18:00.339+0000] {processor.py:161} INFO - Started process (PID=21065) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:00.340+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:00.340+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:00.340+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:00.345+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:00.361+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:00.361+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:18:00.372+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:00.372+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:18:00.395+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.061 seconds
[2024-05-05T14:18:08.544+0000] {processor.py:161} INFO - Started process (PID=21085) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:08.545+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:08.546+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:08.545+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:08.553+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:08.551+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 19, in <module>
    say_hello = py
                ^^
NameError: name 'py' is not defined
[2024-05-05T14:18:08.554+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:08.582+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.045 seconds
[2024-05-05T14:18:10.965+0000] {processor.py:161} INFO - Started process (PID=21164) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:10.966+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:10.968+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:10.967+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:10.978+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:10.974+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 19, in <module>
    say_hello = python
                ^^^^^^
NameError: name 'python' is not defined
[2024-05-05T14:18:10.982+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:11.013+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.059 seconds
[2024-05-05T14:18:14.262+0000] {processor.py:161} INFO - Started process (PID=21192) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:14.263+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:14.264+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:14.263+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:14.276+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:14.270+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 19, in <module>
    say_hello = pythonO
                ^^^^^^^
NameError: name 'pythonO' is not defined
[2024-05-05T14:18:14.279+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:14.308+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.053 seconds
[2024-05-05T14:18:16.610+0000] {processor.py:161} INFO - Started process (PID=21220) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:16.611+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:16.612+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:16.611+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:16.626+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:16.618+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 19, in <module>
    say_hello = pythonOper
                ^^^^^^^^^^
NameError: name 'pythonOper' is not defined
[2024-05-05T14:18:16.635+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:16.662+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.061 seconds
[2024-05-05T14:18:18.725+0000] {processor.py:161} INFO - Started process (PID=21232) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:18.726+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:18.727+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:18.727+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:18.743+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:18.733+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 19, in <module>
    say_hello = pythonOperatoe
                ^^^^^^^^^^^^^^
NameError: name 'pythonOperatoe' is not defined
[2024-05-05T14:18:18.753+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:18.783+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.066 seconds
[2024-05-05T14:18:20.626+0000] {processor.py:161} INFO - Started process (PID=21290) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:20.636+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:20.638+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:20.637+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:20.679+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:20.652+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 19, in <module>
    say_hello = PythonOperatoe
                ^^^^^^^^^^^^^^
NameError: name 'PythonOperatoe' is not defined
[2024-05-05T14:18:20.692+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:20.725+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.114 seconds
[2024-05-05T14:18:22.938+0000] {processor.py:161} INFO - Started process (PID=21310) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:22.939+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:22.940+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:22.940+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:22.956+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:22.946+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 19, in <module>
    say_hello = PythonOperator
                ^^^^^^^^^^^^^^
NameError: name 'PythonOperator' is not defined
[2024-05-05T14:18:22.969+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:23.013+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.085 seconds
[2024-05-05T14:18:31.826+0000] {processor.py:161} INFO - Started process (PID=21372) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:31.827+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:31.828+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:31.828+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:31.845+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:31.839+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 19, in <module>
    say_hello = PythonOperator()
                ^^^^^^^^^^^^^^
NameError: name 'PythonOperator' is not defined
[2024-05-05T14:18:31.850+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:31.875+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.062 seconds
[2024-05-05T14:18:33.845+0000] {processor.py:161} INFO - Started process (PID=21377) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:33.846+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:33.847+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:33.847+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:33.851+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:33.849+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 19
    say_hello = PythonOperator():
                                ^
SyntaxError: invalid syntax
[2024-05-05T14:18:33.851+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:33.877+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.038 seconds
[2024-05-05T14:18:34.911+0000] {processor.py:161} INFO - Started process (PID=21387) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:34.911+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:34.912+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:34.912+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:34.914+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:34.913+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 21
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:18:34.915+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:34.937+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.031 seconds
[2024-05-05T14:18:42.338+0000] {processor.py:161} INFO - Started process (PID=21478) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:42.339+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:42.340+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:42.340+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:42.344+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:42.343+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 22
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:18:42.345+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:42.371+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.039 seconds
[2024-05-05T14:18:45.705+0000] {processor.py:161} INFO - Started process (PID=21514) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:45.705+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:45.706+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:45.706+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:45.708+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:45.707+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 22
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:18:45.709+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:45.730+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.030 seconds
[2024-05-05T14:18:48.955+0000] {processor.py:161} INFO - Started process (PID=21539) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:48.956+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:48.956+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:48.956+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:48.960+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:48.959+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 22
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:18:48.960+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:48.989+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.041 seconds
[2024-05-05T14:18:50.519+0000] {processor.py:161} INFO - Started process (PID=21569) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:50.520+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:50.521+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:50.521+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:50.524+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:50.523+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 22
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:18:50.525+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:50.553+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.044 seconds
[2024-05-05T14:18:54.120+0000] {processor.py:161} INFO - Started process (PID=21619) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:54.120+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:54.121+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:54.121+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:54.124+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:54.123+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 22
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:18:54.124+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:54.146+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.032 seconds
[2024-05-05T14:18:59.741+0000] {processor.py:161} INFO - Started process (PID=21679) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:18:59.742+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:18:59.742+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:59.742+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:18:59.745+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:18:59.744+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 21
    python_callable=,
    ^^^^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-05T14:18:59.745+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:18:59.770+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.035 seconds
[2024-05-05T14:19:02.766+0000] {processor.py:161} INFO - Started process (PID=21684) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:02.767+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:02.767+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:02.767+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:02.770+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:02.769+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 22
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:19:02.770+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:02.795+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.034 seconds
[2024-05-05T14:19:07.843+0000] {processor.py:161} INFO - Started process (PID=21694) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:07.843+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:07.844+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:07.844+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:07.848+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:07.847+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 22
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:19:07.848+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:07.875+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.040 seconds
[2024-05-05T14:19:17.203+0000] {processor.py:161} INFO - Started process (PID=21817) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:17.204+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:17.205+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:17.204+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:17.208+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:17.207+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 22
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:19:17.209+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:17.237+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.041 seconds
[2024-05-05T14:19:39.438+0000] {processor.py:161} INFO - Started process (PID=21984) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:39.439+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:39.441+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:39.441+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:39.444+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:39.443+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 26
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:19:39.445+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:39.471+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.039 seconds
[2024-05-05T14:19:47.138+0000] {processor.py:161} INFO - Started process (PID=22089) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:47.138+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:47.139+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:47.139+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:47.142+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:47.141+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 7
    def _say
            ^
SyntaxError: expected '('
[2024-05-05T14:19:47.143+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:47.170+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.038 seconds
[2024-05-05T14:19:48.780+0000] {processor.py:161} INFO - Started process (PID=22115) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:48.781+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:48.782+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:48.782+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:48.786+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:48.784+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 7
    def _say_hello
                  ^
SyntaxError: expected '('
[2024-05-05T14:19:48.786+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:48.820+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.048 seconds
[2024-05-05T14:19:51.011+0000] {processor.py:161} INFO - Started process (PID=22135) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:51.012+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:51.013+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:51.013+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:51.016+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:51.015+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 7
    def _say_hello()
                    ^
SyntaxError: expected ':'
[2024-05-05T14:19:51.017+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:51.047+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.044 seconds
[2024-05-05T14:19:51.756+0000] {processor.py:161} INFO - Started process (PID=22175) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:51.757+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:51.758+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:51.757+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:51.763+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:51.761+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 10
    with DAG(
    ^^^^
IndentationError: expected an indented block after function definition on line 7
[2024-05-05T14:19:51.764+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:51.801+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.054 seconds
[2024-05-05T14:19:53.929+0000] {processor.py:161} INFO - Started process (PID=22190) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:53.930+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:53.931+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:53.931+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:53.934+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:53.933+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 11
    with DAG(
    ^^^^
IndentationError: expected an indented block after function definition on line 7
[2024-05-05T14:19:53.935+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:53.961+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.039 seconds
[2024-05-05T14:19:56.560+0000] {processor.py:161} INFO - Started process (PID=22245) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:56.561+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:56.562+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:56.561+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:56.565+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:56.564+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:19:56.566+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:56.594+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.041 seconds
[2024-05-05T14:19:58.797+0000] {processor.py:161} INFO - Started process (PID=22270) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:19:58.798+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:19:58.798+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:58.798+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:19:58.801+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:19:58.800+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:19:58.801+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:19:58.822+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.031 seconds
[2024-05-05T14:20:01.834+0000] {processor.py:161} INFO - Started process (PID=22275) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:20:01.835+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:20:01.836+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:20:01.836+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:20:01.840+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:20:01.839+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:20:01.841+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:20:01.874+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.047 seconds
[2024-05-05T14:20:31.968+0000] {processor.py:161} INFO - Started process (PID=22548) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:20:31.968+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:20:31.969+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:20:31.969+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:20:31.971+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:20:31.971+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:20:31.972+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:20:31.993+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.030 seconds
[2024-05-05T14:20:50.253+0000] {processor.py:161} INFO - Started process (PID=22693) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:20:50.254+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:20:50.255+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:20:50.254+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:20:50.258+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:20:50.257+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:20:50.259+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:20:50.289+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.043 seconds
[2024-05-05T14:20:59.227+0000] {processor.py:161} INFO - Started process (PID=22825) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:20:59.228+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:20:59.229+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:20:59.228+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:20:59.231+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:20:59.230+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:20:59.232+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:20:59.254+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.032 seconds
[2024-05-05T14:21:02.252+0000] {processor.py:161} INFO - Started process (PID=22830) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:02.253+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:02.253+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:02.253+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:02.256+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:02.255+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:02.256+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:02.278+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.031 seconds
[2024-05-05T14:21:04.298+0000] {processor.py:161} INFO - Started process (PID=22835) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:04.299+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:04.299+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:04.299+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:04.302+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:04.301+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:04.303+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:04.322+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.031 seconds
[2024-05-05T14:21:07.362+0000] {processor.py:161} INFO - Started process (PID=22845) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:07.362+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:07.363+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:07.363+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:07.366+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:07.365+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:07.367+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:07.392+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.037 seconds
[2024-05-05T14:21:10.431+0000] {processor.py:161} INFO - Started process (PID=22862) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:10.432+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:10.433+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:10.433+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:10.436+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:10.435+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:10.437+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:10.464+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.039 seconds
[2024-05-05T14:21:17.693+0000] {processor.py:161} INFO - Started process (PID=22957) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:17.694+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:17.695+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:17.695+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:17.699+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:17.698+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:17.699+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:17.726+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.040 seconds
[2024-05-05T14:21:18.822+0000] {processor.py:161} INFO - Started process (PID=22972) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:18.822+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:18.823+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:18.823+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:18.827+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:18.825+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:18.828+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:18.854+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.037 seconds
[2024-05-05T14:21:26.456+0000] {processor.py:161} INFO - Started process (PID=23073) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:26.457+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:26.457+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:26.457+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:26.460+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:26.459+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 27
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:26.460+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:26.483+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.032 seconds
[2024-05-05T14:21:33.103+0000] {processor.py:161} INFO - Started process (PID=23133) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:33.104+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:33.105+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:33.104+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:33.107+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:33.106+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 28
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:33.108+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:33.129+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.030 seconds
[2024-05-05T14:21:36.125+0000] {processor.py:161} INFO - Started process (PID=23138) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:36.126+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:36.127+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:36.126+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:36.130+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:36.128+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 28
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:36.130+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:36.151+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.031 seconds
[2024-05-05T14:21:38.191+0000] {processor.py:161} INFO - Started process (PID=23148) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:38.192+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:38.193+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:38.192+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:38.195+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:38.194+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 28
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:38.196+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:38.217+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.031 seconds
[2024-05-05T14:21:41.264+0000] {processor.py:161} INFO - Started process (PID=23164) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:41.265+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:41.266+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:41.266+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:41.269+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:41.268+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 28
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:41.270+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:41.300+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.042 seconds
[2024-05-05T14:21:44.294+0000] {processor.py:161} INFO - Started process (PID=23222) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:44.295+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:44.299+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:44.299+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:44.309+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:44.305+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 28
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:44.310+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:44.338+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.067 seconds
[2024-05-05T14:21:52.414+0000] {processor.py:161} INFO - Started process (PID=23304) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:52.415+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:52.416+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:52.416+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:52.420+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:52.419+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 28
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:52.421+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:52.448+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.040 seconds
[2024-05-05T14:21:56.774+0000] {processor.py:161} INFO - Started process (PID=23375) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:56.774+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:56.775+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:56.775+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:56.778+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:56.777+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 28
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:56.778+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:56.816+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.047 seconds
[2024-05-05T14:21:59.400+0000] {processor.py:161} INFO - Started process (PID=23435) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:21:59.400+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:21:59.401+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:59.401+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:21:59.404+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:21:59.403+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 29
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:21:59.404+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:21:59.428+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.033 seconds
[2024-05-05T14:22:09.491+0000] {processor.py:161} INFO - Started process (PID=23451) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:22:09.492+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:22:09.493+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:09.493+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:22:09.497+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:09.496+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 29
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:22:09.498+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:22:09.524+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.039 seconds
[2024-05-05T14:22:12.638+0000] {processor.py:161} INFO - Started process (PID=23466) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:22:12.640+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:22:12.641+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:12.641+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:22:12.645+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:12.643+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 29
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:22:12.646+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:22:12.680+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.054 seconds
[2024-05-05T14:22:17.074+0000] {processor.py:161} INFO - Started process (PID=23556) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:22:17.074+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:22:17.075+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:17.075+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:22:17.077+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:17.077+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 29
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:22:17.078+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:22:17.101+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.032 seconds
[2024-05-05T14:22:18.100+0000] {processor.py:161} INFO - Started process (PID=23561) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:22:18.102+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:22:18.103+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:18.102+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:22:18.106+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:18.105+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 29
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:22:18.107+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:22:18.150+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.060 seconds
[2024-05-05T14:22:21.583+0000] {processor.py:161} INFO - Started process (PID=23591) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:22:21.584+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:22:21.585+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:21.585+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:22:21.588+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:21.587+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 29
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:22:21.589+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:22:21.625+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.048 seconds
[2024-05-05T14:22:51.856+0000] {processor.py:161} INFO - Started process (PID=23864) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:22:51.858+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:22:51.859+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:51.858+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:22:51.862+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:22:51.861+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 29
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:22:51.863+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:22:51.902+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.052 seconds
[2024-05-05T14:23:22.404+0000] {processor.py:161} INFO - Started process (PID=24136) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:23:22.405+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:23:22.406+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:23:22.406+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:23:22.409+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:23:22.408+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 29
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:23:22.410+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:23:22.439+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.042 seconds
[2024-05-05T14:23:36.417+0000] {processor.py:161} INFO - Started process (PID=24277) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:23:36.418+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:23:36.418+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:23:36.418+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:23:36.421+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:23:36.420+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 30
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:23:36.421+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:23:36.443+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.031 seconds
[2024-05-05T14:24:07.094+0000] {processor.py:161} INFO - Started process (PID=24550) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:24:07.094+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:24:07.095+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:24:07.095+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:24:07.098+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:24:07.097+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 30
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:24:07.098+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:24:07.122+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.033 seconds
[2024-05-05T14:24:11.176+0000] {processor.py:161} INFO - Started process (PID=24567) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:24:11.177+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:24:11.178+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:24:11.178+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:24:11.180+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:24:11.180+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 30
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:24:11.181+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:24:11.203+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.031 seconds
[2024-05-05T14:24:21.360+0000] {processor.py:161} INFO - Started process (PID=24677) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:24:21.361+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:24:21.362+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:24:21.361+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:24:21.364+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:24:21.363+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 30
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:24:21.365+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:24:21.388+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.032 seconds
[2024-05-05T14:24:51.633+0000] {processor.py:161} INFO - Started process (PID=24950) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:24:51.633+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:24:51.634+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:24:51.634+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:24:51.636+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:24:51.636+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 30
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:24:51.637+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:24:51.657+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.030 seconds
[2024-05-05T14:25:10.133+0000] {processor.py:161} INFO - Started process (PID=25113) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:25:10.134+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:25:10.135+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:25:10.135+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:25:10.140+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:25:10.139+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 31
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:25:10.140+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:25:10.170+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.044 seconds
[2024-05-05T14:25:13.220+0000] {processor.py:161} INFO - Started process (PID=25123) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:25:13.220+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:25:13.221+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:25:13.221+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:25:13.224+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:25:13.223+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 31
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:25:13.224+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:25:13.247+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.035 seconds
[2024-05-05T14:25:19.703+0000] {processor.py:161} INFO - Started process (PID=25223) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:25:19.704+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:25:19.705+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:25:19.704+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:25:19.708+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:25:19.707+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 31
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:25:19.709+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:25:19.738+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.042 seconds
[2024-05-05T14:25:49.824+0000] {processor.py:161} INFO - Started process (PID=25495) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:25:49.824+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:25:49.825+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:25:49.825+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:25:49.828+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:25:49.827+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 31
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:25:49.828+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:25:49.850+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.031 seconds
[2024-05-05T14:26:20.088+0000] {processor.py:161} INFO - Started process (PID=25767) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:26:20.088+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:26:20.089+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:26:20.089+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:26:20.097+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:26:20.093+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 31
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:26:20.097+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:26:20.137+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.058 seconds
[2024-05-05T14:26:50.312+0000] {processor.py:161} INFO - Started process (PID=26039) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:26:50.313+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:26:50.314+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:26:50.313+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:26:50.316+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:26:50.315+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 31
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:26:50.317+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:26:50.338+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.031 seconds
[2024-05-05T14:27:21.047+0000] {processor.py:161} INFO - Started process (PID=26311) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:27:21.048+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:27:21.049+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:27:21.048+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:27:21.051+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:27:21.050+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 31
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:27:21.051+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:27:21.072+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.030 seconds
[2024-05-05T14:27:51.138+0000] {processor.py:161} INFO - Started process (PID=26584) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:27:51.139+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:27:51.139+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:27:51.139+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:27:51.142+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:27:51.141+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 31
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:27:51.143+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:27:51.168+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.038 seconds
[2024-05-05T14:28:21.468+0000] {processor.py:161} INFO - Started process (PID=26801) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:28:21.470+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:28:21.471+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:28:21.470+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:28:21.474+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:28:21.473+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hello.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hello.py", line 31
    ):
     ^
SyntaxError: invalid syntax
[2024-05-05T14:28:21.475+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:28:21.509+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.052 seconds
[2024-05-05T14:28:25.490+0000] {processor.py:161} INFO - Started process (PID=26871) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:28:25.491+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:28:25.492+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:28:25.491+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:28:25.500+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:28:25.660+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:28:25.660+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:28:25.677+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:28:25.677+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:28:25.939+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.456 seconds
[2024-05-05T14:28:56.062+0000] {processor.py:161} INFO - Started process (PID=27144) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:28:56.063+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:28:56.064+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:28:56.063+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:28:56.070+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:28:56.094+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:28:56.094+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:28:56.110+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:28:56.110+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:28:56.143+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.088 seconds
[2024-05-05T14:29:26.546+0000] {processor.py:161} INFO - Started process (PID=27416) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:29:26.550+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:29:26.552+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:29:26.552+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:29:26.585+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:29:26.680+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:29:26.679+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:29:26.751+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:29:26.751+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:29:26.952+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.427 seconds
[2024-05-05T14:29:57.827+0000] {processor.py:161} INFO - Started process (PID=27695) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:29:57.828+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:29:57.830+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:29:57.829+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:29:57.836+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:29:57.859+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:29:57.859+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:29:57.876+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:29:57.875+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:29:57.901+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.080 seconds
[2024-05-05T14:30:28.732+0000] {processor.py:161} INFO - Started process (PID=27967) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:30:28.733+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:30:28.734+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:30:28.734+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:30:28.741+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:30:28.764+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:30:28.764+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:30:28.780+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:30:28.780+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:30:28.806+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.080 seconds
[2024-05-05T14:30:59.464+0000] {processor.py:161} INFO - Started process (PID=28240) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:30:59.464+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:30:59.465+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:30:59.465+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:30:59.472+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:30:59.499+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:30:59.498+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:30:59.514+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:30:59.514+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:30:59.542+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.085 seconds
[2024-05-05T14:31:29.919+0000] {processor.py:161} INFO - Started process (PID=28512) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:31:29.920+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:31:29.921+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:31:29.921+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:31:29.927+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:31:29.952+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:31:29.951+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:31:30.135+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:31:30.135+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:31:30.162+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.250 seconds
[2024-05-05T14:32:00.819+0000] {processor.py:161} INFO - Started process (PID=28785) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:32:00.820+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:32:00.821+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:32:00.821+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:32:00.828+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:32:00.854+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:32:00.854+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:32:00.870+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:32:00.870+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:32:00.896+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.085 seconds
[2024-05-05T14:32:31.783+0000] {processor.py:161} INFO - Started process (PID=29058) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:32:31.784+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:32:31.786+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:32:31.785+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:32:31.792+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:32:31.819+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:32:31.819+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:32:31.836+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:32:31.836+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:32:31.863+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.086 seconds
[2024-05-05T14:33:02.297+0000] {processor.py:161} INFO - Started process (PID=29331) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:33:02.298+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:33:02.299+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:33:02.299+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:33:02.308+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:33:02.332+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:33:02.332+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:33:02.349+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:33:02.348+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:33:02.386+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.095 seconds
[2024-05-05T14:33:32.493+0000] {processor.py:161} INFO - Started process (PID=29604) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:33:32.494+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:33:32.495+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:33:32.494+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:33:32.502+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:33:32.526+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:33:32.526+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:33:32.543+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:33:32.543+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:33:32.596+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.110 seconds
[2024-05-05T14:34:03.244+0000] {processor.py:161} INFO - Started process (PID=29877) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:34:03.245+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:34:03.247+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:34:03.246+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:34:03.254+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:34:03.277+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:34:03.277+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:34:03.294+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:34:03.294+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:34:03.320+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.083 seconds
[2024-05-05T14:34:33.917+0000] {processor.py:161} INFO - Started process (PID=30154) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:34:33.918+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:34:33.919+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:34:33.919+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:34:33.926+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:34:33.950+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:34:33.950+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:34:33.967+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:34:33.967+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:34:33.994+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.083 seconds
[2024-05-05T14:35:04.280+0000] {processor.py:161} INFO - Started process (PID=30498) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:35:04.285+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:35:04.293+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:35:04.293+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:35:04.319+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:35:04.569+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:35:04.568+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:35:04.711+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:35:04.711+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:35:04.862+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.611 seconds
[2024-05-05T14:35:35.727+0000] {processor.py:161} INFO - Started process (PID=30852) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:35:35.732+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:35:35.734+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:35:35.734+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:35:35.782+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:35:35.842+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:35:35.842+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:35:35.861+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:35:35.860+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:35:35.921+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.204 seconds
[2024-05-05T14:36:06.351+0000] {processor.py:161} INFO - Started process (PID=31193) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:36:06.352+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:36:06.353+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:36:06.352+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:36:06.359+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:36:06.383+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:36:06.383+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:36:06.405+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:36:06.404+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:36:06.438+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.093 seconds
[2024-05-05T14:36:36.815+0000] {processor.py:161} INFO - Started process (PID=31471) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:36:36.816+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:36:36.817+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:36:36.816+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:36:36.823+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:36:36.864+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:36:36.863+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:36:36.885+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:36:36.884+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:36:36.913+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.111 seconds
[2024-05-05T14:37:07.133+0000] {processor.py:161} INFO - Started process (PID=31748) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:37:07.133+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:37:07.136+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:37:07.135+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:37:07.142+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:37:07.164+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:37:07.164+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:37:07.179+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:37:07.179+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:37:07.206+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.080 seconds
[2024-05-05T14:37:37.394+0000] {processor.py:161} INFO - Started process (PID=32025) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:37:37.394+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:37:37.395+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:37:37.395+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:37:37.402+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:37:37.424+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:37:37.424+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:37:37.440+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:37:37.440+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:37:37.465+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.079 seconds
[2024-05-05T14:38:07.748+0000] {processor.py:161} INFO - Started process (PID=32304) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:38:07.749+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:38:07.750+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:38:07.750+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:38:07.757+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:38:07.782+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:38:07.781+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:38:07.802+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:38:07.802+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:38:07.830+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.088 seconds
[2024-05-05T14:38:38.402+0000] {processor.py:161} INFO - Started process (PID=32581) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:38:38.403+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:38:38.404+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:38:38.404+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:38:38.411+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:38:38.439+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:38:38.438+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:38:38.455+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:38:38.455+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:38:38.484+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.088 seconds
[2024-05-05T14:39:08.741+0000] {processor.py:161} INFO - Started process (PID=32858) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:39:08.742+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:39:08.743+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:39:08.742+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:39:08.749+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:39:08.775+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:39:08.774+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:39:08.792+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:39:08.791+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:39:08.823+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.088 seconds
[2024-05-05T14:39:39.750+0000] {processor.py:161} INFO - Started process (PID=33136) to work on /opt/airflow/dags/hello.py
[2024-05-05T14:39:39.751+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/hello.py for tasks to queue
[2024-05-05T14:39:39.752+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:39:39.752+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/hello.py
[2024-05-05T14:39:39.759+0000] {processor.py:840} INFO - DAG(s) 'hello' retrieved from /opt/airflow/dags/hello.py
[2024-05-05T14:39:39.817+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:39:39.817+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-05T14:39:39.841+0000] {logging_mixin.py:188} INFO - [2024-05-05T14:39:39.840+0000] {dag.py:3954} INFO - Setting next_dagrun for hello to None, run_after=None
[2024-05-05T14:39:39.884+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/hello.py took 0.141 seconds
